### PR TITLE
docs: split local Kubernetes instructions out of installation guide

### PR DESCRIPTION
**Pull request by edvald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can find Garden’s full documentation at [https://docs.garden.io](https://d
 Overview:
 
 - [Basics](https://docs.garden.io/basics)—installation instructions, our quick start guide, and an overview of the main concepts around Garden.
-- [Using Garden](https://docs.garden.io/using-garden)—features and usage, Garden configuration files, usage with remote clusters, and setting up hot reload.
+- [Using Garden](https://docs.garden.io/using-garden)—development workflows, Garden configuration files, usage with remote Kubernetes clusters, and setting up hot reload.
```
Comment (eysi09) [inline README.md]: Not relevant to PR per se but feels like it'll be hard to keep this overview section in sync with the actual docs.

File: docs/basics/installation.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ steps below if you prefer.
 
 If you haven't already set up Homebrew, please follow [their installation instructions](https://brew.sh/).
 
-### Step 2: Install Garden
+### Step 2: Install Garden (macOS)
```
Comment (eysi09) [inline docs/basics/installation.md]: Why add (macOS), it's already under that heading?

File: docs/basics/installation.md [hunk 3]
```diff
@@ -48,23 +48,15 @@ To later upgrade to the latest version, simply run the script again.
 To install Docker, Kubernetes and kubectl, we strongly recommend Docker for Mac. Garden itself doesn't require a local
 installation of Kubernetes, but it is in most cases the preferred way of using it.
```
Comment (eysi09) [inline docs/basics/installation.md]: Is local Kubernetes _in most cases the preferred way of using it_?

File: docs/basics/installation.md [hunk 4]
```diff
@@ -113,94 +105,14 @@ To later upgrade to the latest version, simply run the script again.
...
-        context: minikube
-```
-
-If you happen to have installed both Minikube and a version of Docker for Mac with Kubernetes support enabled,
-`garden` will choose whichever one is configured as the current context in your `kubectl` configuration. If neither
-is set as the current context, the first available context is used.
-
-(If you're not yet familiar with Garden configuration files, see:
-[Configuration files](../using-garden/configuration-files.md))
+If you'd like to use a local Kubernetes cluster, please refer to the [local Kubernetes guide](../using-garden/local-kubernetes.md)
```
Comment (eysi09) [inline docs/basics/installation.md]: This step is missing from the macOS section.

File: README.md (hunk 1 above)
Comment (edvald) [reply README.md]: True. Not sure how best to address that though.

File: docs/basics/installation.md (hunk 2 above)
Comment (edvald) [reply docs/basics/installation.md]: It's to fix a linting warning. There's another heading called "Install Garden" below, which makes it ambiguous.

File: docs/basics/installation.md (hunk 3 above)
Comment (edvald) [reply docs/basics/installation.md]: Nope, I'll update.

**Comment by edvald** [comment]
@eysi09 Oops, force-pushed out of habit. What I updated was just the docker+k8s step for macOS installation.

File: README.md (hunk 1 above)
Comment (eysi09) [reply README.md]: We can revisit later but in general this might be too specific for the README page.

**Pull request merged**